source/changelog.txt

```diff
@@ -12,6 +12,18 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.3.1
+------
+
+*Released September 5, 2024*
+
+Bug fixes in this release:
+
+- :issue:`COMPASS-8252` - Remove certificates without issuer from the TLS CA list
```

Comment:
[question]

Should we also list [MONGOSH-1416](https://jira.mongodb.org/browse/MONGOSH-1416) and [MONGOSH-1859](https://jira.mongodb.org/browse/MONGOSH-1859) here? Thank you!

Reply:
Thanks @jmd-mongo! Great question. I'm basing this off of @addaleax's release summary in the [mongosh](https://mongodb.slack.com/archives/CUHC9R8J0/p1725567397992589) slack channel.

Reply:
I think we can list MONGOSH-1859 here, yeah, as e.g. "`ISODate()` now passes non-string arguments to `new Date()`". 🙂

Reply:
Thanks both and updated!